cmd/ping.go

```diff
@@ -243,14 +243,37 @@ func trimToTwoDecimal(d time.Duration) string {
 	switch {
 	case d >= time.Second:
 		f = float64(d) / float64(time.Second)
-		unit = "s"
+		unit = pad("s", " ", 7-len(fmt.Sprintf("%.02f", f)), false)
 	default:
 		f = float64(d) / float64(time.Millisecond)
-		unit = "ms"
+		unit = pad("ms", " ", 6-len(fmt.Sprintf("%.02f", f)), false)
 	}
 	return fmt.Sprintf("%.02f%s", f, unit)
 }
 
+func pad(s, p string, count int, left bool) string {
```

Comment:
Some comments about what this is doing would be good - I can't understand what is being done here.

Reply:
done